src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-conditions.mdx

```diff
@@ -0,0 +1,394 @@
+---
+title: Alert conditions
```

Comment:
This title could convey what the doc has you do. i think 'Modify alerts with conditions' or something to that effect that is more active/action oriented